src/components/Table/_table.scss

```diff
@@ -1 +1,23 @@
 @import 'TableHead/table-head';
+@import '~carbon-components/scss/components/data-table/data-table';
```

Comment:
This import can be removed, Table relies on DataTable, which includes this import in it's sass partial